_includes/manuals/3.1/1.2_release_notes.md

```diff
@@ -31,6 +31,13 @@ A new cron job has been introduced to purge the old Audits on a daily basis. The
 #### Provisioning snippets support Puppet 7
 The provisioning snippets have been updated to setup Puppet 7 repositories, as well as configure and install packages from those repositories.
 
+#### Performance improvements for index pages and Host Config Status
```

Comment:
@ekohl or @ezr-ondrej request to review this point. I think maybe I have added more details than required?

Reply:
For a headline feature it does have a lot of details. I'd try to keep it shorter and leave out some implementation details. Focus on the results for the end user.